examples/test/utils/helpers.js

```diff
@@ -120,6 +120,25 @@ export const loginUsingLoginForm = (redirectUri = "/", retryCount = 2) => async
     );
 };
 
+export const regularUser = Role(`${config.url}`, async (tc = testController) => {
+    // wait till s-isWaitingForLoggedInStatus disappears
+    // allow long timeout because of page load
+    await retry(() => selectorExists(".s-isWaitingForLoggedInStatus", false), 15000).catch(error => {
+        // eslint-disable-next-line no-console
+        console.error("ERROR: s-isWaitingForLoggedInStatus forever. Probably a JS issue", error);
+        // no reason to retry, something is most likely broken
+    });
+
+    await tc
+        .typeText(".s-login-input-email", config.username, { paste: true, replace: true })
+        .typeText(".s-login-input-password", config.password, { paste: true, replace: true })
+        .click(".s-login-submit");
+});
+
+export const loginUser = (redirectUri = "/") => async (tc = testController) => {
```

Comment:
The name is confusing to me, it logs the user in and then redirects them somewhere. What about somthing like `loginAndNavigateTo`?